.rubocop_todo.yml

```diff
@@ -114,7 +114,7 @@ RSpec/ExampleLength:
 # Offense count: 2
 # Configuration parameters: Include, CustomTransform, IgnoreMethods, SpecSuffixOnly.
 # Include: **/*_spec*rb*, **/spec/**/*
-RSpec/FilePath:
+RSpec/SpecFilePathSuffix:
```

Comment:
Please fix these instead by renaming the files to have a `_spec.rb` suffix. The cop tells you the tests in those files aren't executed right now.